<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLWindowLevelWidget</class>
 <widget class="QWidget" name="qMRMLWindowLevelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="WindowLevelLabel">
     <property name="text">
      <string>Window/Level: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QComboBox" name="AutoManualComboBox">
     <item>
      <property name="text">
       <string>Manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Auto</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDoubleSpinBox" name="WindowSpinBox">
     <property name="prefix">
      <string>W: </string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="ctkDoubleRangeSlider" name="WindowLevelRangeSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="LevelSpinBox">
     <property name="prefix">
      <string>L: </string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QToolButton" name="MoreOptionsButton">
     <property name="icon">
      <iconset resource="../qMRMLWidgets.qrc">
       <normaloff>:/Icons/SliceMoreOptions.png</normaloff>:/Icons/SliceMoreOptions.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDoubleRangeSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleRangeSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
